content/es/docs/templates/feature-state-alpha.txt

```diff
@@ -0,0 +1,7 @@
+La funcionalidad está actualmente en *alpha*:
+
+* El nombre de la versión contiene alpha (por ejemplo, v1alpha1)
+* No está completamente testeada, por lo que habilitar la funcionalidad puede exponer problemas. Se encuentra deshabilitada por defecto.
+* El soporte de la funcionalidad puede desestimarse en cualquier momento sin previo aviso.
+* La API puede romper la compatibilidad en releases posteriores sin previo aviso.
+* Se recomienda utilizar este tipo de funcionalidades unicamente en clústeres efímeros para testing, dado el elevado riesgo de problemas que puede provocar y la falta de soporte a larga plazo.
```

Comment:
- tilde en únicamente.
- to diría probada en vez de testeada, pero fuera de es-ES tiene sentido.

Reply:
Fixed en https://github.com/kubernetes/website/compare/59443f001cb2b6d99a1ae7e2da7c1e40c99caf79..8dd0998ed991350def717719bd77a0b295e1e24a